File: PRStest/PRStest/Product.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace PRStest
{
    public class Product
    {
        public int ID { get; set; }
        public virtual Vendor VendorId { get; set; }
        [StringLength(50)]
        [Required]
        public string PartNumber { get; set; }
        [StringLength(150)]
        [Required]
        public string Name { get; set; }
        [Required]
        public decimal Price { get; set; }
        [StringLength(255)]
        [Required]
        public string Unit { get; set; }
        [StringLength(255)]
        public string PhotoPath { get; set; }
        [Required]
        public bool Active { get; set; }
        public DateTime? DateCreated { get; set; }
        public DateTime? DateUpdated { get; set; }
    }
}

File: PRSmaster/PRSweb-master/PRSweb-master/ng/src/app/models/Product.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var Product = /** @class */ (function () {
    function Product(Id, VendorPartNumber, Name, Price, Unit, PhotoPath, VendorId) {
        this.Id = Id;
        this.VendorPartNumber = VendorPartNumber;
        this.Name = Name;
        this.Price = Price;
        this.Unit = Unit;
        this.PhotoPath = PhotoPath;
        this.VendorId = VendorId;
    }
    return Product;
}());
exports.Product = Product;
